kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -133,7 +133,8 @@
   }
 
   public static ConfigDef newConfigDef() {
-    ConfigDef configDef = StorageSinkConnectorConfig.newConfigDef(FORMAT_CLASS_RECOMMENDER);
+    ConfigDef configDef = StorageSinkConnectorConfig.newConfigDef(FORMAT_CLASS_RECOMMENDER,
```

Comment:
Multi-line calls need each argument in its own line, as: 
```java
ConfigDef configDef = StorageSinkConnectorConfig.newConfigDef(
   FORMAT_CLASS_RECOMMENDER,
   AVRO_COMPRESSION_RECOMMENDER
);
```

Reply:
Thanks, @kkonstantine. Fixed this.